jena-text/src/test/java/org/apache/jena/query/text/TestTextIndexES.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jena.query.text;
+
+
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.vocabulary.RDFS;
+import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsRequest;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+/**
+ *
+ * Integration test for {@link TextIndexES} class
+ * ES Integration test depends on security policies that may sometime not be loaded properly.
+ * If you find any issues regarding security set the following VM argument to resolve the issue:
+ * -Dtests.security.manager=false
+ *
+ */
+@ESIntegTestCase.ClusterScope()
+public class TestTextIndexES extends ESIntegTestCase {
```

Comment:
Also, it would be good to have a unit test that tests for language tag subcodes, e.g. add an entity like
`:col rdfs:label "color"@en-US, "colour"@en-GB`
then test that 
- it can be found using either `color` or `colour`, without lang parameter
- it can be found using either `color` or `colour` using `lang:en*` as parameter
- it can be found using `color` `lang:en-US`
- it cannot be found using `color` `lang:en-GB`